### DISCUSSION TITLE
Filters in Media Management

### DISCUSSION THREAD

**Discussion by daeks** [gotson/komga#1802] [Ideas]
Hi,

first of all, thank you very much for this nice piece of software. I would like to suggest some additions to the media management section:

- Some more filters, esp. for the libraries - It is quite nasty if you have multiple libraries containing errors to have some overview
- Showing the stacktrace as details in errors like "Unknown error while analyzing book"

**Comment by gotson** [comment]
I've added filters for status and libraries
<img width="1184" alt="image" src="https://github.com/user-attachments/assets/e055c353-e1f7-4095-a131-7ee1d19788d5" />

**Comment by gotson** [comment]
🎉 This issue has been resolved in `1.19.0` ([Release Notes](https://github.com/gotson/komga/releases/tag/1.19.0))

**Comment by daeks** [comment]
thank you very much :) as the media management is now splitted up into multiple pages, can this be added also to "Missing poster" and " duplicate files/pages"?
